snowflake-api/src/responses.rs

```diff
@@ -69,6 +70,7 @@ macro_rules! into_resp_type {
             code: $base_res.code.clone(),
             message: $base_res.message.clone(),
             success: $base_res.success,
+            query_id: $base_res.query_id.clone(),
```

Comment:
Since this `query_id` does not appear good enough (being reassigned later), would it be safer to set it to `None` here?